tests/trace_server/test_remote_http_trace_server.py

```diff
@@ -1,127 +0,0 @@
-import datetime
```

Comment:
do we not longer want to test error code handling?

Reply:
not sure why this got deleted; undeleted